dev/AppNotifications/ShellLocalization.h

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "pch.h"
+
+// In this file we define methods that write/return Shell assets, like DisplayName and icon URI.
+
+namespace Microsoft::Windows::AppNotifications::ShellLocalization
+{
+    const std::wstring c_appIdentifierPath{ LR"(Software\Classes\AppUserModelId\)" };
+    const std::wstring c_quote{ LR"(")" };
+    const std::wstring c_backSlash{ LR"(\)" };
```

Comment:
Why? If this is just used to do string concatenation of paths you should use instead std::filesystem::path